cypress/e2e/types/constants.ts

```diff
@@ -224,3 +224,11 @@ export enum TaskKind {
     techDiscovery = "tech-discovery",
     languageDiscovery = "language-discovery",
 }
+
+export enum taskFilter {
```

Comment:
We are naming all enums starting in uppercase `TaskFilter`

```suggestion
export enum TaskFilter {
```